dmoj/executors/gcc_executor.py

```diff
@@ -22,7 +23,7 @@
 
 class GCCExecutor(CompiledExecutor):
     defines = []
-    flags = []
+    flags = ['-fmax-errors=%d' % MAX_ERRORS]
```

Comment:
This feels like an awful idea. The whole point of `flags` here is so that subclasses can override it. We don't actually have a metaclass to inherit by concatenation, nor can we expect subclasses to write `flags = GCCExecutor.flags + [...]`.